css/templates/lune/components/_line-after.css

```diff
@@ -1,3 +1,13 @@
 /* //////////////////////////////////////////////////////
   LINE AFTER COMPONENT
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+/* To be used in conjunction with bass css 'flex items-center' classes */
+.lune-line-after::after {
```

Comment:
We intentionally made this into a component as it's used outside of the products page (On the store page too)